conf/modules.config

```diff
@@ -11,17 +11,12 @@
 ----------------------------------------------------------------------------------------
 */
 
-// Generic process options for all workflows
+// CUSTOM_DUMPSOFTWAREVERSIONS
 process {
-
-    publishDir = [
```

Comment:
Maybe lets keep the default publishing strategy, or did you experience difficulties with it?